WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -220,8 +220,8 @@ public static PSBT ParsePsbt(string json, Network network)
 
 	public static HwiEnumerateEntry ParseHwiEnumerateEntry(JObject json)
 	{
-		JToken modelToken = json["model"];
-		var pathString = json["path"]?.ToString()?.Trim();
+		JToken modelToken = json["model"]!;
+		var pathString = json["path"]!.ToString().Trim();
```

Comment:
Same as before.

Reply:
But we have this:
```
	public static IEnumerable<HwiEnumerateEntry> ParseHwiEnumerateResponse(string responseString)
	{
		var jarr = JArray.Parse(responseString);

		var response = new List<HwiEnumerateEntry>();
		foreach (JObject json in jarr)
		{
			var hwiEntry = ParseHwiEnumerateEntry(json);
			response.Add(hwiEntry);
		}

		return response;
	}
```
If there is no HW connected, `jarr` will be empty anyway, no call there, but if there **is** one, i.e. a TrezorT, `modelToken` can't be null, am I not right?

Reply:
@Szpoti this is not about HWs, this is about an external input that can be null. Software have to verify external inputs or at least do not assume they will be correct because they are out of the program's control.

In this case `json["model"]` and `json["path"]` can be `null`.

Btw, if you want to check values you can try with the `TryGet` or similar. Alternatively you can create an extension method `GetOrFail` or something smart.